jobs/cloud_controller_ng/spec

```diff
@@ -376,6 +376,8 @@ properties:
     description: "When enabled, the priority of asynchronous jobs will be increased by 1 for each active asynchronous job for that user. 
                   This prevents that a single user, who creates many asynchronous jobs, slows down job execution for other users."
     default: false
+  cc.jobs.threads:
```

Comment:
I'm somehow missing the information that this is a number, i.e. the number of threads.